contents/docs/product-analytics/cutting-costs.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Cutting costs
+sidebar: Docs
+showTitle: true
+---
+
+We aim to be significantly cheaper than our competitors. In addition to our [ridiculously cheap pricing](/pricing), below are tips to reduce your product analytics costs:
```

Comment:
Hmm you raise a valid point. Im leaning to either removing the intro completely, or leaving as is. Reasoning:

Someone is probably looking at this page because their PostHog costs are too high. By saying "In addition to our ridiculously cheap pricing", it's probably rubbing salt in their wound.

On the other hand, they're probably wondering if competitors are cheaper if their PostHog costs are high, so it's good to reassure them that they're not. Actually I htink we should leave it in because of this

(sorry for thinking out loud lol)